src/main/java/de/ellpeck/actuallyadditions/mod/blocks/BlockLaserRelay.java

```diff
@@ -259,10 +260,10 @@ public void displayHud(Minecraft minecraft, EntityPlayer player, ItemStack stack
 
                         String expl;
                         if(compass){
-                            expl = relay.getCompassDisplayString();
+                            expl = relay.getCompassDisplayString().replaceAll("\\\\n", "\n");
                         }
                         else{
-                            expl = TextFormatting.GRAY.toString()+TextFormatting.ITALIC+"Hold a Compass to modify!";
+                            expl = TextFormatting.GRAY.toString()+TextFormatting.ITALIC+StringUtil.localize("info."+ModUtil.MOD_ID+".laserRelay.mode.noCompasss").replaceAll("\\\\n", "\n");
```

Comment:
Due to how minecraft handles localization, strings are formatted multiple times, so a "\" each time the string is formatted, I already encoundered that problème before and there no better way to handle it.
And yeah I totally forgot to remove it :p